docs/core-tags.md

```diff
@@ -1,434 +1,388 @@
 >
 > ```marko
 > <await(slowPromise) timeout=5000>
->    <@then>Done</@then>
->    <@catch|err|>
->      <if(err.name === "TimeoutError")>
->        Took too long to fetch the data!
->      </if>
->      <else>
->        Promise failed with ${err.message}.
->      </else>
->    </@catch>
+>   <@then>Done</@then>
+>   <@catch|err|>
+>     <if(err.name === "TimeoutError")>
+>       Took too long to fetch the data!
+>     </if>
+>     <else>
+>       Promise failed with ${err.message}.
+>     </else>
+>   </@catch>
 > </await>
 > ```
 
 ## `<include-text>`
 
-You can inline abritrary text files into a template using the `include-text` tag.
-Special HTML characters will be escaped. If you do not want escaping then use the `<include-html>` tag (see below).
+`<include-text>` inlines text files into a template, escaping HTML syntax characters (`<`, `"`, etc.).
 
 ```marko
 <include-text('./foo.txt')/>
 ```
 
+If you do not want escaping, use [`<include-html>`](#include-html) instead.
+
 ## `<include-html>`
 
-Like the `include-text` tag, the `include-html` tag allows you to inline the contents of a file. However this tag does _not_ escape special HTML characters.
+Like `<include-text>`, `<include-html>` inlines the contents of a file. However, this tag **does _not_ escape** special HTML characters.
 
 ```marko
 <include-html('./foo.html')/>
 ```
 
 ## `<html-comment>`
 
-Standard HTML comments are automatically stripped from the output with Marko. For cases where you need to have these comments included with the output you can leverage the `<html-comment>` tag.
-
-_Marko Source:_
+Marko removes HTML comment tags from its output. But if you need comments in the output, that’s what `<html-comment>` is for:
 
 ```marko
-<html-comment>[if IE]><script src="..."></script><![endif]</html-comment>
+<html-comment>[if IE]><script src="html-shiv.js"></script><![endif]</html-comment>
 ```
 
-_HTML Output:_
+…becomes:
 
 ```html
-<!--[if IE]><script src="..."></script><![endif]-->
+<!--[if IE]><script src="html-shiv.js"></script><![endif]-->
 ```
 
-> **Note:** You may also see the deprecated `<marko-compiler-options>` tag used to configure comments for the entire template:
+> **Note:** You might see the **deprecated** `<marko-compiler-options>` tag used to configure comments for the template:
 >
 > ```marko
 > <marko-compiler-options preserve-comments/>
 > ```
 
 ## Deprecated
 
-The following tags and attributes are deprecated, but you may see them in existing code.
+The following tags and attributes are deprecated, but you might see them in older code.
 
 ### `marko-preserve-whitespace`
 
-Whitespace can be preserved using the `preserve-whitespace` attribute:
+Instead, preserve whitespace with the `preserve-whitespace` attribute:
 
 ```marko
-<div marko-preserve-whitespace>
-    All of this
-    whitespace   will
-    be preserved.
-</div>
-```
+<pre marko-preserve-whitespace>
```

Comment:
Since `<pre>` will preserve whitespace by default, `marko-preserve-whitespace` is redundant here.  But `<div>` alone also isn't a great example because preserving the whitespace isn't going to matter.  Perhaps we should include a style block that gives the tag `white-space:pre`:

```marko
style {
    .poem {
        white-space: pre;
    }
}

<div.poem>
    who ate all the cookies??
                   I     did.
</div>
```

Reply:
Excellent point. I’ve been trying to think of an example where `marko-preserve-whitespace` would fit well on a known, semantic use-case, but it looks like all the elements that want that behavior have it built-in (`<pre>`, `<textarea>`, `<script>`, etc.)

I don’t want to use a `<div>` if only because concrete poems, semantically, _should_ be `<pre>` — if Marko didn’t discourage generation of XML (#1398), something involving `xml:space="preserve"` might work…

Other examples I can think of similarly shouldn’t be encouraged — `<xmp>`, `<plaintext>`, `<listing>`…

I know this seems trivial, but I think docs are one of the best ways to teach/encourage good HTML, so I care about it maybe more than I should. What use-cases was `marko-preserve-whitespace` added for in the first place?

Reply:
I propose we punt on this because without a real use-case for why `marko-preserve-whitespace` exists, I’m not sure the docs backing it up is that important?

Reply:
Yeah, I'd like to get the rest of the changes in... 

What about this as an example? 
```html
<style>
  code {
    white-space: pre;
  }
</style>

<code marko-preserve-whitespace>
  import { foo } from "bar";
  console.log(foo);
</code>
```